sdk/identity/identity/src/client/errors.ts

```diff
@@ -149,9 +149,19 @@ export class AggregateAuthenticationError extends Error {
    */
   public errors: any[];
 
-  constructor(errors: any[]) {
+  constructor(errors: any[], errMsg?: string) {
     super(
-      `Authentication failed to complete due to the following errors:\n\n${errors.join("\n\n")}`
+      `${errMsg ? errMsg : ""}\n\n${
+        errors
+          .toString()
```

Comment:
I find relying on `toString()` to join an Array a bit magical. `errors.join(",")` is more clear